### ISSUE TITLE
identical compare breaks old radio checked option

### ISSUE THREAD

**Issue by m3di** [LaravelCollective/html#538]
https://github.com/LaravelCollective/html/blob/7c781e74fbc9e95f7c67636ed6728040dae23b44/src/FormBuilder.php#L871

because of the identical compare the old submitted key casts into string and no longer returns true while comparing the key.

just replace the compare with ==

i know this is fixed in 5.6 but i need this with php7.0  😪😪😪

**Comment by jshah4517** [comment]
We have this same issue as we've just upgraded to Laravel 5.5, is it possible this fix can be backported?

**Comment by Vinze** [comment]
I agree, Laravel 5.5 is an LTS release so it would be nice to have this fix backported.

**Comment by wturrell** [comment]
[Here's the diff of the fix](https://github.com/LaravelCollective/html/commit/5060a6641dbdf72ea277e30f353931da2617c00a), for anyone like me who's recently done composer update on a Laravel 5.5 LTS app and happened across it. 

Note there are *two* methods that need changing, `getCheckedState()` and `getRadioCheckedState()`
